### DISCUSSION TITLE
I just updated my index file few hours ago and there's this error

### DISCUSSION THREAD

**Discussion by SoumyaK4** [cotes2020/jekyll-theme-chirpy#1982] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

[Error link](https://github.com/SoumyaK4/weiqi/actions/runs/11158953376)

For fixing this do I have to do something in local environment?
I tried some other repo in local environment before, one problem I faced was that in local it works perfectly but on GitHub htmlproofer gives errors, though most are fixed with config file, some errors I just have no idea why.

### Operations you have already tried

Just updated my gemfile but still same.

**Comment by kungfux** [comment]
> [Error link](https://github.com/SoumyaK4/weiqi/actions/runs/11158953376)

This page does not exist. It would be better to describe the problem here so it can be useful for others in the future.

**Reply by SoumyaK4** [reply, marked as answer]
I actually redid the whole website.
But you actually answered in another post that this happened because I upgraded the chirpy version, but I had too much custom codes.
